common/src/main/java/com/wynntils/mc/mixin/PlayerTabOverlayMixin.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.mc.mixin;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.components.Managers;
+import com.wynntils.features.ui.PlayerInfoFeature;
+import net.minecraft.client.gui.components.PlayerTabOverlay;
+import net.minecraft.world.scores.Objective;
+import net.minecraft.world.scores.Scoreboard;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(PlayerTabOverlay.class)
+public class PlayerTabOverlayMixin {
+
+    @Inject(method = "render", at = @At("HEAD"), cancellable = true)
+    private void render(PoseStack poseStack, int width, Scoreboard scoreboard, Objective objective, CallbackInfo ci) {
+        if (Managers.Feature.getFeatureInstance(PlayerInfoFeature.class).isEnabled()) ci.cancel();
```

Comment:
This is not how we do this. Check out what `com.wynntils.mc.event.RenderEvent.ElementType` is and how it is used to replace GUI elements by other overlays.